src/translations/en.json

```
@@ -262,21 +262,21 @@
   "SwitchedOnMaxQAtNominalV": "Switch-on Q at nominal voltage",
```

"maxQAtNominalV": "Available Qmax at nominal voltage (MVar)",
```suggestion
  "SwitchedOnMaxQAtNominalV": "Switch-on Q at nominal voltage (MVar)",
```

---

Done